src/main/java/seedu/address/logic/commands/EditItemCommand.java

```diff
@@ -35,9 +36,11 @@
             + "[" + PREFIX_ORIGINAL_ITEM_NAME + "NAME] "
             + "[" + PREFIX_ITEM_NAME + "NAME] "
             + "[" + PREFIX_ITEM_QUANTITY + "QUANTITY] "
-            + "[" + PREFIX_ITEM_DESCRIPTION + "DESCRIPTION] \n"
+            + "[" + PREFIX_ITEM_DESCRIPTION + "DESCRIPTION] "
+            + "[" + PREFIX_ITEM_TAG + "TAG/TAGS] "
             + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_ITEM_QUANTITY + "20 ";
+            + PREFIX_ITEM_QUANTITY + "20 "
+            + PREFIX_ITEM_TAG + "tag1, tag2,tag3 , tag4";
```

Comment:
spacing after `,`

Reply:
this is intentional to show user that comma spacing is irrelevent.